pkg/controller/integrationplatform/initialize.go

```diff
@@ -115,19 +115,29 @@ func (action *initializeAction) Handle(ctx context.Context, platform *v1alpha1.I
 		return err
 	}
 
+	if target.Spec.Build.PublishStrategy == v1alpha1.IntegrationPlatformBuildPublishStrategyKaniko {
+		// create and warm the Kaniko cache into the Camel K builder volume
+		action.L.Info("Warming Kaniko cache...")
+		err = warmKanikoCache(ctx, action.client, target)
```

Comment:
wondering if this warming up could be translate into an additional phase and the warmKanikoCache done in a separate co-routine to avoid blocking the reconcile loop

Reply:
Yes, totally make sense! Let me rework that part.

Reply:
Done